src/cms/data/functions/seedData.js

```diff
@@ -28,6 +28,7 @@ const loadData = async () => {
       otherData.loadUrgency(),
       otherData.loadFireCentre(),
       otherData.loadFireZone(),
+      otherData.loadRedirects(),
```

Comment:
If we are going to load redirects as part of the data import, then they should also be deleted and loaded in **reimportData.js**.  

Add these lines where you see `park-name` being deleted and reloaded:
`otherData.loadRedirects(),`
`strapi.services["redirects"].delete(),`